roxie/ccd/ccdfile.cpp

```diff
@@ -1852,16 +1863,23 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
 
     void doLoadSavedOsCacheInfo(unsigned channel)
     {
-        const char* dllserver_root = getenv("HPCC_DLLSERVER_PATH");
-#ifdef _CONTAINERIZED
-        assertex(dllserver_root != nullptr);
-#else
-        //Default behaviour is to not load or saving anything on bare metal
-        if (!dllserver_root)
-            return;
-#endif
+        StringBuffer cacheRootDirectory;
+        if (isContainerized())
+        {
+            if (!getConfigurationDirectory(nullptr, "query", nullptr, nullptr, cacheRootDirectory))
+                throwUnexpected();
+        }
+        else
+        {
+            //Default behaviour is to not load or saving anything on bare metal
```

Comment:
Either this comment is wrong, or the previous code that asserted the env var was set is wrong.

Reply:
The assert was if containerized was true - this is if it is false.

Reply:
Same comment re env var name as before